web/packages/teleport/src/services/user/user.ts

```diff
@@ -60,13 +61,24 @@ const service = {
     return api.put(cfg.getUsersUrl(), user).then(makeUser);
   },
 
-  createUser(user: User) {
-    return api.post(cfg.getUsersUrl(), user).then(makeUser);
+  createUser(user: User, webauthnResponse?: WebauthnAssertionResponse) {
+    return api
+      .post(cfg.getUsersUrl(), user, null, webauthnResponse)
+      .then(makeUser);
   },
 
-  createResetPasswordToken(name: string, type: ResetPasswordType) {
+  createResetPasswordToken(
+    name: string,
+    type: ResetPasswordType,
+    webauthnResponse?: WebauthnAssertionResponse
+  ) {
     return api
-      .post(cfg.api.resetPasswordTokenPath, { name, type })
+      .post(
+        cfg.api.resetPasswordTokenPath,
+        { name, type },
+        null,
+        webauthnResponse
+      )
       .then(makeResetToken);
   },
```

Comment:
Hmm I don't see these new parameters being used, they also don't compile.

Reply:
I'll have a follow up PR where they are used. I'll fix the compiling issue

Reply:
I'm not sure why/when this wasn't compiling, but it is working for me currently.